librz/core/casm.c

```diff
@@ -705,16 +705,7 @@ static RzList *rz_core_asm_back_disassemble (RzCore *core, ut64 addr, int len, u
 			last_num_invalid = 0;
 		// disassembly underlap
 		} else if (current_buf_pos + current_instr_len < next_buf_pos) {
-			ut32 purge_results = 0;
```

Comment:
why deleted everything here? I think those functions have side effects that cannot be removed.